test/test_built_in_functions.py

```diff
@@ -21,6 +21,38 @@ def test_array_assign(self):
         self.assertEqual('arr[x] = 6', main.main('arr[x] = 6', 'js', 'py'))
         self.assertEqual('array[3] = []', main.main('array[3] = []', 'py', 'js')) 
     
+    def test_sort(self):
+        self.assertEqual('myArr.sort()', main.main('myArr.sort()', 'js', 'py'))
```

Comment:
What if the sort method is used on a non-array, like for example a custom class that happens to also have that method?

Reply:
I think we get burned in that case. But we discussed this and the plan is to punt on that for now, it's a pretty big/ambiguous task and there may be lower hanging fruit.

With that said @cjoshea9 can we create a ticket for this? As we add more types users are more likely to run into these cases.

Reply:
Yes, we decided to punt on that for now, but I will make a ticket to address this later. @g-harel @augerm